### DISCUSSION TITLE
Facebook Stream Stop on reconnect rtmp stream from software to antmedia

### DISCUSSION THREAD

**Discussion by mariia-artamonova** [ant-media/Ant-Media-Server#5947] [FAQs] [answered]
hello, i have a little problem creating a stream on facebook using a software that sends facebook data to antmedia and then creates an rtmp stream in antmedia and uses that facebook endpoint and creates the stream there, but in the application i have a section which does the rtmp reconnect, which does rtmp.close, and then reconfigures and starts it, antmedia receives it and works fine with youtube for example, but when I create the stream with facebook, it ends it when I do that reconnect, but in antmedia continues as streaming, maybe there is some parameter in the server that keeps the stream without ending it unless I end it manually in antmedia?

here are my logs on PasteBin:
https://pastebin.com/gFsC4cGP

**Comment by yashtandon113** [comment, marked as answer]
Hi @mariia-artamonova 
I have tested the scenario by adding the Facebook RTMP endpoint, and it works fine for me even after reconnecting again. Here is the reproduce scenario:

1. Create a live stream on the server
2. Add FB rtmp endpoint
3. Publish RTMP stream on the server (Stream is republished to the Facebook end point)
4. Close the stream after some time
5. Reconnect the stream again (Stream is republished again to the Facebook end point without any issue.)

Please make sure that you are using the latest version, 2.7.0, of Ant Media Server.
